### ISSUE TITLE
Visualization confidence socre very low

### ISSUE THREAD

**Issue by laplaceson** [zylo117/Yet-Another-EfficientDet-Pytorch#650]
Hi, thanks to your great work.
I try to train on my customize dataset and Pascal VOC's TV class.
Both of them get very low score output. (Ex 0.10, 0.11 etc.)
Do you have any idea where might be the problem?
Thanks! ( Btw, all of these two training are single class)

**Comment by laplaceson** [comment]
My training image is get from Pascal VOC 's tv class.(And split into 400 image train,69 test)
Here's the result print using debug = true.
![0c37fb4cd64943ce92bd22af91a7809b](https://user-images.githubusercontent.com/20430525/118913825-c77e3900-b95c-11eb-8ad8-c22344a3f48f.jpg)

**Comment by zylo117** [comment]
Keep training and it should get better

**Comment by laplaceson** [comment]
Thanks for reply!
Like this size of dataset (400 for train, 69 for test). How many epoch would you suggest?
Now I am training to 200 epoch. But still get similar result.
And the loss seems can not goes down anymore.
![image](https://user-images.githubusercontent.com/20430525/118941110-fa87f300-b983-11eb-858c-79aacb47e2c9.png)
I am using head_only = True with D0 model and pretrained weight.
Do you have any suggestion on this situation? 
Thanks

**Comment by zylo117** [comment]
400? You shouldn't expect any good result coming out of it.
Can you share your training command?

**Comment by laplaceson** [comment]
Thanks for noticing, I'm just want to verify at least it can fit to training data.
Here's my command 
CUDA_VISIBLE_DEVICES=1,2 python train.py 

and i modify the parser
head only to True
using -c 0
load_weights efficientdet-d0.pth
lr 1e-3
debug True
batch_size 16
num_workers 0

**Comment by zylo117** [comment]
It seems ok. So you should change `head_only` to False and keep training.

**Comment by laplaceson** [comment]
Thank you very much ! I will follow the suggestion
